src/token.test.ts

```diff
@@ -0,0 +1,38 @@
+import Token from ".";
+import { tokenRopstenValid } from "../test/fixtures/tokenRopstenValid";
+/* eslint-disable global-require */
+jest.mock("./util/provider", () => {
```

Comment:
```suggestion
jest.mock("./util/provider", () => ({
  getProvider: require("ganache-cli").provider
}))
```

Reply:
Is there a reason you are using require instead of import also?

Reply:
not particularly just one-liner code. dont need to import and use.